bot/__main__.py

```diff
@@ -1,43 +1,45 @@
 import os
 
-from discord import Game
 from discord.ext.commands import when_mentioned_or
 
-from bot import config
-from bot.core.bot import Bot
+import config
+from core.bot import Bot
 
 TOKEN = os.getenv("BOT_TOKEN")
+PREFIX = config.COMMAND_PREFIX
 
 extensions = [
     "bot.cogs.codesandbox",
+    "bot.cogs.coding",
     "bot.cogs.commands",
-    "bot.cogs.conversion",
+
+    "bot.cogs.comics",
     "bot.cogs.common",
+    "bot.cogs.conversion",
+    "bot.cogs.documentation",
+    "bot.cogs.embeds",
     "bot.cogs.emotes",
     "bot.cogs.events",
     "bot.cogs.fun",
     "bot.cogs.games",
+    "bot.cogs.github",
     "bot.cogs.moderation",
+    "bot.cogs.nasa",
+    "bot.cogs.reddit",
+    "bot.cogs.search",
+    "bot.cogs.security",
     "bot.cogs.study",
     "bot.cogs.sudo",
     "bot.cogs.support",
     "bot.cogs.tools",
-    "bot.cogs.search",
-    "bot.cogs.security",
-    "bot.cogs.embeds",
-    "bot.cogs.comics",
-    "bot.cogs.coding",
-    "bot.cogs.documentation",
-    "bot.cogs.reddit",
     "bot.cogs.translate",
-    "bot.cogs.github",
-    "bot.cogs.nasa",
 ]
 
+
 bot = Bot(
     extensions,
-    command_prefix=when_mentioned_or(config.COMMAND_PREFIX),
-    activity=Game(name=f"Ping me using {config.COMMAND_PREFIX}help"),
+    PREFIX,
+    when_mentioned_or(PREFIX),
```

Comment:
There is no reason to do this, we can simply use the keyword argument `command_prefix=...` what improvement does this serve?

Reply:
That is because the first one is used in the status message, whereas the second one is the real command_prefix
I prefer it that way, because if we setup some custom prefix function, we'll be able to pass it straight into command_prefix without having to edit the status message

Reply:
However, if you want, I can fetch it straight from the config file if you prefer

Reply:
Ok fixed that my bad.